tox.ini

```diff
@@ -1,20 +1,13 @@
 [tox]
-envlist = {py27}-dj{14,15,16,17}-rsms{13,14,15,16,17,18,19}
+envlist = {py27}-dj{17,18}-rsms{18,19}
```

Comment:
Might be nice to add environments for flake8 and coverage, but certainly not needed in this PR.